packages/apps/graph/src/graph/query/events.ts

```diff
@@ -99,7 +99,7 @@ builder.queryField('events', (t) =>
             }),
           },
           orderBy: {
-            id: 'desc',
+            height: 'desc',
```

Comment:
Ordering by just `height` might cause subtle issues since `height`-alone doesn't determine a total order. I think it's best to order by `(height, requestkey, idx)`.